SPECS/kernel/config

```diff
@@ -544,6 +544,7 @@ CONFIG_CPU_SRSO=y
 # CONFIG_GDS_FORCE_MITIGATION is not set
 CONFIG_MITIGATION_RFDS=y
 CONFIG_MITIGATION_SPECTRE_BHI=y
+CONFIG_MITIGATION_ITS=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/c5a5d8075231e59aa2ed8b3a2e80698f068a39d8